### ISSUE TITLE
libXGL_DPP is constantly printing messages

### ISSUE THREAD

**Issue by MarkRivers** [epics-modules/Dante#8]
When the Dante IOC is running with the 3.7.8.0 SDK there is a constant stream of messages on the EPICS IOC console:

```
Identifier: 164.54.160.186
Identifier: 164.54.160.186
Identifier: 164.54.160.186
Identifier: 164.54.160.186
Identifier: 164.54.160.186
Identifier: 164.54.160.186
Identifier: 164.54.160.186
Identifier: 164.54.160.186
Identifier: 164.54.160.186
Identifier: 164.54.160.186
Identifier: 164.54.160.186
Identifier: 164.54.160.186
```
These are not coming from the EPICS code, so they must be coming from the libXGL_DPP library.  They interfere with using the IOC console.  Can they be turned off?

**Comment by lucagrittiniXGLAB** [comment]
Hi Mark,
this issue has been solved in the library I committed today (3.7.13.0)

**Comment by MarkRivers** [comment]
This is verified to be fixed in the new library.
